diff --git a/third_party/blink/renderer/modules/sanitizer_api/sanitizer.h b/third_party/blink/renderer/modules/sanitizer_api/sanitizer.h
index 3d3144ccdb8d6640c2554f64c2e4c281bc5a1ffe..15ee37de2e1b14680a4ce6c7dc5f480bb409a6fa 100644
--- a/third_party/blink/renderer/modules/sanitizer_api/sanitizer.h
+++ b/third_party/blink/renderer/modules/sanitizer_api/sanitizer.h
@@ -13,28 +13,18 @@ namespace blink {
 
 class DocumentFragment;
 class ExceptionState;
-class SanitizerConfig;
 class ScriptState;
 
 class MODULES_EXPORT Sanitizer final : public ScriptWrappable {
   DEFINE_WRAPPERTYPEINFO();
 
  public:
-  static Sanitizer* Create(const SanitizerConfig*, ExceptionState&);
-  explicit Sanitizer(const SanitizerConfig*);
+  static Sanitizer* Create(ExceptionState&);
+  Sanitizer();
   ~Sanitizer() override;
 
   String sanitizeToString(ScriptState*, const String&, ExceptionState&);
   DocumentFragment* sanitize(ScriptState*, const String&, ExceptionState&);
-
-  SanitizerConfig* creationOptions() const;
-
-  void Trace(Visitor*) const override;
-
- private:
-  // TODO(lyf): Make config_ read-only. The creationOptions getter which
-  // asks for the pointer is forbidened by a read-only variable.
-  Member<SanitizerConfig> config_ = {};
 };
 
 }  // namespace blink
